pkg/proxy/proxy.go

```diff
@@ -348,6 +348,9 @@ func BootProxy(logger *zap.Logger, opt Option, appCmd, appContainer string, pid
 	if opt.Port == 0 {
 		opt.Port = 16789
 	}
+	for !isPortAvailable(opt.Port) {
```

Comment:
Define upper and lower bounds. For loop running continuously is not preferred, define max attempts.